rpa-bot/1.3/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -28,12 +30,28 @@ include::rpa-home::partial$windows-versions-support.adoc[]
 
 |===
 
-=== Requirements for Secure Session Technology
+=== System Settings
+
+* Display resolution: 
+** Minimum: 1280 x 1080
+** Recommended: Full HD 1080p (1920 × 1080) or higher
+* Scaling of text and other items on the screen: 100 %
+* _Clear Type_: turned off
+
+=== Requirements for the Secure Session Technology
+
+Secure Sessions are hidden Desktop sessions. They run in the background and only the bot can access them. An additional advantage of using Secure Sessions is, that the bot can multiply itsself to run several process instances in parallel.
+
+To use Secure Sessions you must ensure that all prerequisites are met and all responsible persons are informed.
```

Comment:
```suggestion
Before you implement Secure Sessions, first meet all prerequisites and inform all responsible persons:
```

Reply:
The person reading this can not meet all the prerequisites and might not know who the responsible persons are. I think this is part of the support problem... 
With that text we can at least raise awareness I hope.

Reply:
What about:
Before you implement Secure Sessions, first ensure all prerequisites are met and inform all responsible persons: